apps/src/lib/jsdebugger/JsDebugger.jsx

```diff
@@ -143,10 +152,60 @@ var UnconnectedJsDebugger = React.createClass({
 
   getInitialState() {
     return {
-      watchersHidden: false
+      watchersHidden: false,
+      open: true,
     };
   },
 
+  componentDidMount() {
+    this.props.debuggerUi.initializeAfterDomCreated({
+      defaultStepSpeed: this.props.stepSpeed,
+      root: this.root,
+      component: this,
+    });
+  },
+
+  isOpen() {
+    return this.state.open;
+  },
+
+  slideShut() {
+    const closedHeight = $(this.root).find('#debug-area-header').height() + $(this.root).find('#debugResizeBar').height();
+    this.setState({
+      transitionType: 'closing',
+      open: false,
+      openedHeight: $(this.root).height(),
+      closedHeight,
+    });
+    $('#codeTextbox').animate(
```

Comment:
For cases like this, where we're modifying portions of the DOM that we don't own, it might be worth a short comment explaining where this comes from.

I wonder if we could/should grab a reference to `#codeTextbox` on mount, comment there, and then elsewhere do something like `this.codeTextbox`.